capture_frames.py

```diff
@@ -1,13 +1,16 @@
 from __future__ import print_function
-import numpy as np
+import os
+import argparse
 import datetime
+import numpy as np
```

Comment:
Alphabetical order pls.

You can check the retrain file to see how that files does it which I think is quite nice.